### DISCUSSION TITLE
How to switch git remote repository addresses with visual actions

### DISCUSSION THREAD

**Discussion by Sallyfafafa** [eclipse-theia/theia#12202] [Ideas]
Vscode does support changing the git remote repository address through the interface.
I also want to be able to clone projects directly when theia opens an empty folder.

**Comment by vince-fugnitto** [comment]
@Sallyfafafa thank you for the question, do you mind sharing some additional details, which feature exactly allowed you to change the remote repository address through the interface in vscode?

If the feature is directly present in vscode it is likely that you can make use of the builtin [`vscode.git`](https://open-vsx.org/extension/vscode/git) and [`vscode.git-base`](https://open-vsx.org/extension/vscode/git-base) plugins for git support, else you may need to extend `@theia/git` to implement the feature yourself.
